packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseOperationObject-test.js

```diff
@@ -389,4 +378,88 @@ describe('Operation Object', () => {
       });
     });
   });
+
+  describe('#responses', () => {
```

Comment:
To be honest, this specific test case has nothing to do with this PR. Except I was debugging some behaviour related and noticed there wasn't a test for responses.